### ISSUE TITLE
public Path then, browser url can not found 

### ISSUE THREAD

**Issue by Shin-Beomchul** [module-federation/module-federation-examples#1938]
TL;DR

working 💯  : http://localhost:8082
working 💯 : router.push({ path: "/userinfo" });
**not Working** 👎  : Enter the url directly into the browser : http://localhost:8082/userinfo 

**Problem** 
![image](https://user-images.githubusercontent.com/25116734/165012872-0ade92fe-a49d-4abb-8347-c7cc21e4fafa.png)

**work**
The first execution path works normally.
 http://localhost:8082/  :  it good working 

**what I want** 
 I want to go immediately to a specific page on the remote with a browser url( http://localhost:8082/userinfo)

**Problem Step** : 
1. git pull : https://github.com/module-federation/module-federation-examples/tree/master/vue3-cli-demo/
2. Create UserInfo.vue page & Add router  in app-exposes(Remote)
3. browser url write and Enter :  http://localhost:8082/userinfo
4. error..! can not found userinfo.vue router page 
![image](https://user-images.githubusercontent.com/25116734/165012570-7083b1fa-a78b-4927-84f2-d0ac78311d70.png)

Maybe it's a problem with publicPath: "auto", I earnestly pray that a savior will appear to solve my problems.

**Comment by Shin-Beomchul** [comment]
solved

**Comment by eliranAharoni** [comment]
@Shin-Beomchul can you please share what was the solution?
